developers_chamber/ecs_utils.py

```diff
@@ -449,6 +450,29 @@ def get_tasks_for_service(cluster, service, region):
     return tasks_arns
 
 
+def stop_services_and_wait_for_tasks_to_stop(cluster, services, timeout, region):
+    ecs_client = _get_ecs_client(region)
+
+    THREAD_MAX = 32
+    number_of_services = len(services)
+
+    tasks = []
+    pool = ThreadPool(number_of_services if number_of_services < THREAD_MAX else THREAD_MAX)
+
+    for service in services:
+        service_tasks = get_tasks_for_service(cluster=cluster, service=service, region=region)
+        if not service_tasks:
+            LOGGER.info("No active tasks found in service '{}'".format(service))
+            continue
+
+        tasks += service_tasks
+        stop_service(cluster=cluster, service=service, region=region)
```

Comment:
takže stopování je synchronní ale čekání asynchronní, ok.

Reply:
teoreticky jsi obojí mohl udělat v tom threadu

Reply:
vím, ale pak je výpis na přeskáčku a je to trochu zmatené, tak jsem to radši udělal takhle pro přehlednost :)

Reply:
navíc to stop_service je strašně rychlý :)